test/rule-yo-version.js

```diff
@@ -0,0 +1,23 @@
+'use strict';
+var assert = require('assert');
+var rule = require('../lib/rules/yo-version');
+
+describe('yo version', function () {
+  it('pass if it\'s new enough', function (cb) {
+    rule.OLDEST_YO_VERSION = 'v0.0.0';
```

Comment:
This is a bad pattern as it could easily break in the future.

The correct way to test this would be to actually mock `latestVersion`. You can do this using proxyquire.